volume-group-operator/controllers/utils/volumegroupcontent.go

```diff
@@ -64,6 +59,15 @@ func (r *ControllerUtils) generateVolumeGroupContentStatus(groupCreationTime *me
 	}
 }
 
+func (r *ControllerUtils) UpdateVolumeGroupStatus(logger logr.Logger, vgc *volumegroupv1.VolumeGroupContent, groupCreationTime *metav1.Time, ready *bool) error {
```

Comment:
can you move `UpdateVolumeGroupStatus` function above `generateVolumeGroupContentStatus` function?

Reply:
done